AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2471,7 +2461,7 @@ - (void)__setSupernode:(ASDisplayNode *)newSupernode
             
             // Propagate down the new pending transition id
             ASDisplayNodePerformBlockOnEverySubnode(self, NO, ^(ASDisplayNode * _Nonnull node) {
-              node.pendingTransitionID = _pendingTransitionID;
+              node.pendingTransitionID = pendingTransitionId;
```

Comment:
This was just a minor performance get, by using the local variable rather than the instance variable.